### ISSUE TITLE
Convert bidi Showcase demos

### ISSUE THREAD

**Issue by anovstrup** [scalagwt/scalagwt-sample#21]
Bidirectional input and formatting (under i18n)

**Comment by gkossakowski** [comment]
Isn't it done already?

**Comment by anovstrup** [comment]
No, these demos didn't exist when we originally forked the Showcase. See http://gwt.google.com/samples/Showcase/Showcase.html#!CwBidiInput and http://gwt.google.com/samples/Showcase/Showcase.html#!CwBidiFormatting.
